columnflow/columnar_util.py

```diff
@@ -6,14 +6,7 @@
 
 from __future__ import annotations
 
-__all__ = [
-    "mandatory_coffea_columns", "ColumnCollection", "EMPTY_INT", "EMPTY_FLOAT",
-    "Route", "RouteFilter", "ArrayFunction", "TaskArrayFunction", "ChunkedIOHandler",
-    "eval_item", "get_ak_routes", "has_ak_column", "set_ak_column", "remove_ak_column",
-    "add_ak_alias", "add_ak_aliases", "update_ak_array", "flatten_ak_array", "sort_ak_fields",
-    "sorted_ak_to_parquet", "sorted_ak_to_root", "attach_behavior", "layout_ak_array",
-    "flat_np_view", "ak_copy", "fill_hist", "deferred_column", "optional_column",
-]
+__all__ = []
```

Comment:
Having all of them listed in `__all__` would only be helpful if one does `from columnflow.columnar_util import *`. Since this is anyway discouraged, we can empty the list (fyi, not defining `__all__ = []` would result in `from columnflow.columnar_util import *` importing the entire local module scope ...).